esp/src/eclwatch/DynamicESDLDetailsWidget.js

```diff
@@ -260,7 +267,7 @@ define([
             var currSel = this.getSelectedChild();
             if (currSel.id == this.widget._Summary.id && !this.widget._Summary.__hpcc_initalized) {
                 this.widget._Summary.__hpcc_initalized = true;
-                var table = domConstruct.create("table", {});
+                var table = domConstruct.create("table", {id: this.id + "_serviceInformation"});
```

Comment:
I added it because it differentiates the start of a new item. However, I do agree "_" should be reserved for tabs. Removed.